projects/storefrontlib/src/lib/cart/facade/cart.service.ts

```diff
@@ -1,18 +1,21 @@
 import { Injectable } from '@angular/core';
+
 import { Store, select } from '@ngrx/store';
-import { filter } from 'rxjs/operators';
+
 import { Observable } from 'rxjs';
+import { filter } from 'rxjs/operators';
 
-import * as fromReducer from '../store/reducers';
 import * as fromAction from '../store/actions';
+import * as fromReducer from '../store/reducers';
 import * as fromSelector from '../store/selectors';
-
-import * as fromAuthSelectors from './../../auth/store/selectors';
+import { AuthService } from '../../auth/index';
 
 import { ANONYMOUS_USERID, CartDataService } from './cart-data.service';
 
 @Injectable()
 export class CartService {
+  callback: Function;
```

Comment:
this one should be 'private' or 'protected', right?

Reply:
Done.